meta-lmp-base/recipes-graphics/wayland/weston_9.0.0.bbappend

```diff
@@ -0,0 +1,9 @@
+# Weston with Xwayland support (requires X11 and Wayland)
+PACKAGECONFIG[xwayland] = "-Dxwayland=true,-Dxwayland=false,libxcursor"
+
+PACKAGECONFIG_remove_apalis-imx6 = "kms"
+
+# Clients support
+SIMPLE_CLIENTS = "all"
+SIMPLE_CLIENTS_apalis-imx6 = "damage,im,egl,shm,touch,dmabuf-v4l"
```

Comment:
Might need a comment here describing why we're using a smaller subset of simple-clients for apalis-imx6

Reply:
Maybe move this change to a file like: `meta-lmp-bsp/recipes-graphics/wayland/weston_%.bbappend` where we can land SoC specific config changes.  @ricardosalveti this isn't quite like a PACKAGECONFIG change like below.. what do you think?